generators/languages/generator.mjs

```diff
@@ -328,13 +328,13 @@ export default class LanguagesGenerator extends BaseApplicationGenerator {
     const generator = _this || this;
     const prefix = this.fetchFromInstalledJHipster('languages/templates');
     const langJavaProp = languageToJavaLanguage(lang);
-    generator.renderTemplate(
+    generator.writeFile(
```

Comment:
Can it be renamed writeTemplate if it is able to interpolate templates?

Reply:
writeFile matches the writeFiles.
IMO writeFile is ok.